packages/components/src/styles/form-styles.ts

```diff
@@ -132,14 +133,17 @@ export const getLabelStyles = (
 
   return {
     label: {
-      display: 'flex',
-      flexDirection: 'column',
-      gap: spacingStaticXSmall,
```

Comment:
Hmm, how does grid improve stuff? Also `gap` appears to be lost?

Reply:
We can position also layered elements like spinner easier and properly without the need for custom calculations to center something e.g. above the checkbox input.